atlp-rwanda/e-comm-team-amigos-fn
# #184948248 Setup GitHub actions

#### What does this PR do?
This PR configures GitHub actions for continuous integration
#### Description of Task to be completed?
- configure GitHub actions
#### How should this be manually tested?
- clone this repo `git clone https://github.com/atlp-rwanda/e-comm-team-amigos-fn`
- switch your working directory `cd e-comm-team-amigos-fn`
- install all dependencies by running `npm install`
- make any relevant change and push
- the tests should run automatically after a push
#### Any background context you want to provide?
- Node JS installed in your local machine
#### What are the relevant pivotal tracker stories?
Story ID: [#184948248](https://www.pivotaltracker.com/story/show/184948248)
